### PR TITLE
[ADD] website_cms: Fronted tests for creation, search and edition on CMS pages.

### PR DESCRIPTION
**Pull request by oscarolar**
WIP

### CODE REVIEW

File: website_cms/templates/assets.xml [hunk 1]
```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
```
Comment (leemannd) [inline website_cms/templates/assets.xml]: `<odoo>` without `<data>` is better

File: website_cms_search/templates/assets.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```
Comment (leemannd) [inline website_cms_search/templates/assets.xml]: prefer odoo instead of `<openerp> <data>`

**Comment by leemannd** [comment]
:+1:

**Comment by oscarolar** [comment]
@pedrobaeza we need to enable code coverage in this repo, can you help us with this?

**Pull request merged**